tests/forge/MixedSwapCallParameters.t.sol

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {Test, stdJson, console2} from "forge-std/Test.sol";
+import {ERC20} from "solmate/src/tokens/ERC20.sol";
+import {ERC721} from "solmate/src/tokens/ERC721.sol";
+import {UniversalRouter} from "universal-router/UniversalRouter.sol";
+import {Permit2} from "permit2/src/Permit2.sol";
+import {DeployRouter} from "./utils/DeployRouter.sol";
+import {MethodParameters, Interop} from "./utils/Interop.sol";
+
+contract SwapERC20CallParametersTest is Test, Interop, DeployRouter {
+    using stdJson for string;
+
+    address private constant RECIPIENT = 0xaAaAaAaaAaAaAaaAaAAAAAAAAaaaAaAaAaaAaaAa;
+    ERC20 private constant WETH = ERC20(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+    ERC20 private constant USDC = ERC20(0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48);
+    ERC20 private constant DAI = ERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
```

Comment:
These token addresses are also in uniswapData.ts - could be worth having a common import instead of hardcoding them in both?

Reply:
Or even import here from `uniswapData.ts`?